docs/proposals/kubelet-qos.md

```diff
@@ -0,0 +1,142 @@
+# Quality of Service in Kubernetes

```

Comment:
I'll rename it to Resource Quality of Service - in our docs resource refers to memory/CPU resources instead of API resource (e.g. docs/design/resources.md). @pmorie if you have strong objections, let me know! :)